.gitignore

```diff
@@ -1 +1,3 @@
 __pycache__
+venv/
```

Comment:
I think neither of these are required? The plugin doesn't require a separate venv (it's running in Kodi anyway), and there is no need to build a ZIP (see below as well).

Reply:
this is so i could install the kodi stubs to get some code completion from VS Code realting to the Kodi functions. I prefer to use a venv to do this rather than installing it locally/globally to avoid polluting my users python libaries with development libaries

Reply:
If you want to keep them locally, and ignored. You can add it to `.git/info/exclude`. Git also reads that file as a `.gitignore` but it will be local only / not part of the repository.

Reply:
> from VS Code

That explains all the (re)formatting of further unchanged code. PyCharm doesn't do that (automatically), which I prefer.